pkgs/tools/misc/ripgrep-all/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, makeWrapper, ffmpeg,
+  pandoc, poppler_utils }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ripgrep-all";
+  version = "0.9.2";
+
+  src = fetchFromGitHub {
+    owner = "phiresky";
+    repo = pname;
+    rev = version;
+    sha256 = "1knv0gpanrid9i9mxg7zwqh9igdksp1623wl9iwmysiyaajlbif2";
+  };
+
+  cargoSha256 = "0xwsx0x9n766bxamhnpzibrnvnqsxz3wh1f0rj29kbl32xl8yyfg";
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/rga \
+      --prefix PATH ":" "${lib.makeBinPath [ ffmpeg pandoc poppler_utils ]}"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "rga: ripgrep, but also search in PDFs, E-Books, Office documents, zip, tar.gz, etc.";
```

Comment:
Please start with capital letter and remove the trailing dot.

Reply:
I can start with the capital letter, but I won't remove the trailing dot as _etc_ is an abbreviation.

Reply:
In that case please either change the description or make sure you don't check that it is compatible with CONTRIBUTING.md when it is not.

Reply:
Okay, I changed it to not have a trailing dot. But mind that `CONTRIBUTING.md` clearly states to *not have a period at the end*. That is not a period, that is a dot because *etc* is an abbreviation.